index.html

```diff
@@ -34,5 +34,296 @@ <h1>I’m Alexandra. <br />Glad to see you!</h1>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <button class="btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="cards card-3">
+                <div>
+                    <img src="/assets/images/image3.svg" alt="screenshot" class="screenshot">
+                </div>
+                <div class="card-description">
+                    <div>
+                        <h3 class="card-title">Tonic</h3>
+                        <div class="card-bg-info">
+                            <p>Canopy</p>
+                            <ul>
+                                <li>Back End Dev</li>
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="card-text">
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="card-skills">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <button class="btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="cards card-4">
+                <div>
+                    <img src="/assets/images/image4.svg" alt="screenshot" class="screenshot">
+                </div>
+                <div class="card-description">
+                    <div>
+                        <h3 class="card-title">Multi-Post Stories</h3>
+                        <div class="card-bg-info">
+                            <p>Canopy</p>
+                            <ul>
+                                <li>Back End Dev</li>
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="card-text">
+                        <p>
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="card-skills">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <button class="btn">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Kindly try to match your design to the Figma design. The background of the work section is not matched the Figma design. Also, there is a horizontal scrollbar on your design. kindly fix the issue. :+1: 
[horizontal scrollbar.webm](https://user-images.githubusercontent.com/70483835/227731205-4f28679a-886d-493f-b746-bb913ee32d57.webm)

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![yourdesign_work](https://user-images.githubusercontent.com/70483835/227731056-ccfcb32c-679a-48a9-8a57-6e6acc2d067c.png) | ![figma_work](https://user-images.githubusercontent.com/70483835/227731054-83935597-e679-48e7-b92e-c6e9fbb9b564.png)

Reply:
This issue is corrected too